data/src/main/java/me/androidbox/data/mapper/ExtensionMapper.kt

```diff
@@ -28,7 +47,7 @@ fun Event.toEventEntity(): EventEntity {
         remindAt = this.remindAt,
         eventCreatorId = this.eventCreatorId,
         isUserEventCreator = this.isUserEventCreator,
-        attendees = this.attendees,
-        photos = this.photos
+        attendees = attendeeConverter.toJson(this.attendees),
+        photos = this.photos.toJson()
     )
```

Comment:
Before inserting the event into the room db, the list of attendees and list of photos will be serialized to json.